index.js

```diff
@@ -301,29 +314,34 @@ module.exports = {
       runtime = null,
       eventType = null,
       repoUrl = null,
+      organization = null,
     } = activityParameters;
 
     const repositoryUrl = repoUrl ? repoUrl : remoteFromLocalGitRepoUrl;
+    const {
+      name: repoName,
+      owner: repoOrganization,
+    } = extractGitHubRepoData(repositoryUrl);
 
-    const repoName = module.exports.extractGitHubRepoPath(repositoryUrl);
     const changelog = await module.exports.generateChangelog(
       commitId,
-      service ? service : repoName,
+      service ?? repoName,
```

Comment:
We cannot support this operators in order to be compatible with Node versions older than 14